mocha.css

```diff
@@ -212,10 +220,12 @@ body {
   text-align: center;
   background: #eee;
   font-size: 15px;
-  -moz-border-radius: 15px;
-  border-radius: 15px;
-  -webkit-transition: opacity 200ms;
-  -moz-transition: opacity 200ms;
+  -webkit-border-radius: 15px;
+     -moz-border-radius: 15px;
+          border-radius: 15px;
+  -webkit-transition:opacity 200ms;
+  -moz-transition:opacity 200ms;
+  -o-transition:opacity 200ms;
```

Comment:
At first, I didn't know why the hell that one prefix was hanging out there either; I was probably as surprised as you that it wasn't my mistake.😂